coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/UrlRep.scala

```diff
@@ -12,15 +12,15 @@ class UrlRep(val base: String) extends RepUtils with SqlPKableColumnRep[SoQLType
 
   val physColumns: Array[String] = Array(base + "_url", base + "_description")
 
-  override val keyColumns: Array[String] = Array(physColumns(urlOffset))
-
-  val sqlTypes: Array[String] = Array("TEXT", "TEXT")
-
   // sub-column offsets
   val urlOffset = 0
   val descriptionOffset = 1
   val lastOffset = 2
 
+  override val keyColumns: Array[String] = Array(physColumns(urlOffset))
+
+  val sqlTypes: Array[String] = Array("TEXT", "TEXT")
```

Comment:
Because the `physColumns(urlOffset)` bit in the first line refers to `urlOffset`, which wasn't initialized yet.  It only ever worked by accident (the default value for an int is 0) and scala 2.12 warns about it.

Reply:
But what about the `sqlTypes` bit?

Reply:
I actually moved the three integers.  It's just the diff choosing to show the other thing because it makes for a slightly smaller diff - three lines instead of four - and it can't tell what I actually did.